packages/models-library/src/models_library/services.py

```diff
@@ -373,6 +374,12 @@ class ServiceDockerData(ServiceKeyVersion, ServiceCommonData):
         ..., description="definition of the outputs of this node"
     )
 
+    boot_options: Optional[BootOptions] = Field(
+        None,
+        alias="boot-options",
```

Comment:
so this one goes on the image right? this one is correct.

Reply:
yes, this is for the image and it is correct